src/CatalogCrawler/Reports/LatestCatalogLeafByPackageReportUpdater.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Versioning;
+
+namespace Knapcode.CatalogCrawler
+{
+    class LatestCatalogLeafByPackageVisitor : ICsvAggregateReportUpdater<PackageIdentityKey, LatestCatalogLeafByPackage>
```

Comment:
I followed the pattern you have in your repo where the type and file names are different (`XVisitor` is in a `XReportUpdate.cs` file). I'm guessing that naming scheme wasn't intentional. Please let me know if you want me to rename all the types or files to something

Reply:
Oh yeah that was definitely a typo. I should probably look for an analyzer to catch these things 😅